setup.py

```diff
@@ -34,7 +34,7 @@
     # https://github.com/virtuald/pyhcl/pull/57
     'pyhcl~=0.3',
     'six',
-    'typing',
+    'typing;python_version<"3.5"',
```

Comment:
Should be `<"3.6"`, right?

Reply:
> NOTE: in Python 3.5 and later, the typing module lives in the stdlib, and installing this package has NO EFFECT. To get a newer version of the typing module in Python 3.5 or later, you have to upgrade to a newer Python (bugfix) version.

https://pypi.org/project/typing/

Reply:
Ok, you're right, I think it was one version later -- I think the Pipfile ^ can be updated then to match.

Reply:
updated Pipfile to match